nextcord/ext/commands/bot.py

```diff
@@ -694,7 +694,10 @@ def _load_from_module_spec(
 
         extras = extras or {}
         try:
-            setup(self, **extras)
+            if asyncio.iscoroutinefunction(setup):
+                asyncio.create_task(setup(self, **extras))
```

Comment:
This will error if no loop exists, a case which is fairly familiar on bot startup. Please catch this exception and provide some form of helpful information, such as an FAQ entry or something along the lines of call `load_ext..` once your bot is running or has an event loop.
A FAQ entry is likely better as you can show example ways to load things correctly.
```py
async def main():
    bot.load_ext...
    await bot.start()

asyncio.run(main())
```

Reply:
seems like a better solution than mine...

Reply:
![image](https://user-images.githubusercontent.com/70831061/159034965-f11bdb71-dd44-49c1-a000-94320cc1885b.png)
@Skelmis looks like ooliver disagreed with catching error

Reply:
Or use asyncio.get_event_loop which I believe creates a new loop if it doesn't exist, and you can catch the error and create a new loop yourself if that works too

Reply:
Psst: *don't* do this. `asyncio` as a whole discourages against making loops yourself unless you're *really* low level. As of Python 3.10, `asyncio.get_event_loop` is deprecated and will no longer make the event loop from 3.11+, making it an alias to `get_running_loop`.

Not catching this exception makes sense since there's no need, but there should also be a FAQ entry about it too.

Reply:
We shouldn't be creating a loop every time we load a cog..? The event loop for the program relates to when the bot is ran

Reply:
@ooliver1 @Astrea49 @Skelmis how do you think about using `asyncio.new_event_loop()` or make a new helper method names `nextcord.utils.get_event_loop`?

Reply:
> @ooliver1 @Astrea49 @Skelmis how do you think about using `asyncio.new_event_loop()` or make a new helper method names `nextcord.utils.get_event_loop`?

Don't do either. Let loading fail if there's no loop - that's honestly ideal. Add an FAQ entry about it instead.

Reply:
My personal opinion is no. We shouldn't 'just make' a new event loop because it's a cheap fix to a problem rather then actually documenting in say, an faq, how to use your new feature.

I can't find things on mobile in the source itself, but we'd be creating atleast in my case up to 10 event loops which is absurd. 
The bot runs on a single event loop as is standard and we should be using that loop.

Reply:
I agree with the comment above mine.